filament/src/components/RenderableManager.cpp

```diff
@@ -375,7 +375,7 @@ void FRenderableManager::create(
             }
         }
 
-        if (builder->mMorphingEnabled) {
+        if (UTILS_UNLIKELY(count > 0 || builder->mMorphingEnabled)) {
```

Comment:
We should rename `count` to `boneCount` because it used very far from its definition.  Also a comment would be helpful to explain why this is necessary even when morphing is disabled.

Reply:
Yes that's good point. It's updated.